src/shared/pages/list/entityList/entityDefinitions/ukviEnquiryReason/__tests__/enquiryReasonView.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { match, MemoryRouter } from 'react-router-dom';
+import { createBrowserHistory, History, Location } from 'history';
+import { act, wait, render, RenderResult } from '@testing-library/react';
+import EntityListView from '../../../entityListView';
+import * as ListService from '../../../../../../services/entityListService';
+import { State } from '../../../state';
+import * as useError from '../../../../../../hooks/useError';
+import enquiryReason from '../ukviEnquiryReason';
+
+let match: match<any>;
+let history: History<any>;
+let location: Location;
+let mockState: State;
+
+const CampaignsView = EntityListView(enquiryReason);
```

Comment:
That old one again, can the var name be updated to reference Enq reason please

Reply:
This has been corrected.